packages/core/modern.config.ts

```diff
@@ -5,21 +5,6 @@ import { tailwindConfig } from './tailwind.config';
 const require = createRequire(import.meta.url);
 const tailwindPlugin = require('@modern-js/plugin-tailwindcss').default;
 
-const COMMON_EXTERNALS = [
```

Comment:
No need to external these packages?

Reply:
'runtime' is bundleless, so this can be moved, in fact, it is only used in `@rspress/theme-default`.
I guess these lines are migrated from modern doc, because 'theme-default' was inside 'core' before.